src/main/cms/cms.c

```diff
@@ -344,10 +346,15 @@ static int cmsDrawMenuItemValue(displayPort_t *pDisplay, char *buff, uint8_t row
     return cnt;
 }
 
+#ifdef USE_OSD
+static int cmsDrawMenuEntry(displayPort_t *pDisplay, OSD_Entry *p, uint8_t row, uint8_t cursor_row)
```

Comment:
It may be better to just pass generic `int param`, so that function signature is not conditional.

And maybe pass `bool selected` / `bool hasCursor` instead of `cursor_row`

Reply:
Done.